app/depict/src/main/java/org/openscience/cdk/depict/Depiction.java

```diff
@@ -0,0 +1,322 @@
+/* Copyright (C) 2015  The Chemistry Development Kit (CDK) project
```

Comment:
Feel free to say here:

Copyright (C) 2015  NextMove Software Ltd <https://nextmovesoftware.com>